### PR TITLE
Dell.DisplayManager version 2.0

### PR DESCRIPTION
**Pull request by DanHSO**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/82170)

### CODE REVIEW

File: manifests/d/Dell/DisplayManager/2.0/Dell.DisplayManager.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Dell.DisplayManager
+PackageVersion: 2.0
+PackageLocale: en-US
+Publisher: EnTech Taiwan
```
Comment (ItzLevvie) [inline manifests/d/Dell/DisplayManager/2.0/Dell.DisplayManager.locale.en-US.yaml:7]:
```suggestion
Publisher: Dell Inc.
```

File: manifests/d/Dell/DisplayManager/2.0/Dell.DisplayManager.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://dl.dell.com/FOLDER09021993M/1/ddmsetup.exe
+  InstallerSha256: 59436F0E170B41A53E777CFC48E80E7C624E7DDA1161B12BB250B9CDD0CE70A6
+  InstallerSwitches:
+    InstallLocation: /DIR=<INSTALLPATH>
+  UpgradeBehavior: uninstallPrevious
+  ProductCode: '{AC50C05D-9D57-40F5-B2EF-AC402F14312B}_is1'
```
Comment (ItzLevvie) [inline manifests/d/Dell/DisplayManager/2.0/Dell.DisplayManager.installer.yaml:17]:
```suggestion
  ProductCode: 'Dell Display Manager 2.0'
  AppsAndFeaturesEntries:
  - DisplayName: Dell Display Manager 2.0
    DisplayVersion: 2.0.0.134
    ProductCode: Dell Display Manager 2.0
```

File: manifests/d/Dell/DisplayManager/2.0/Dell.DisplayManager.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  DisplayInstallWarnings: false
+  ReleaseDate: 2022-09-26
+ManifestType: installer
+ManifestVersion: 1.2.0
+ReleaseDate: 2022-02-14
```
Comment (ItzLevvie) [inline manifests/d/Dell/DisplayManager/2.0/Dell.DisplayManager.installer.yaml:22]:
```suggestion
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please sign the CLA: https://github.com/microsoft/winget-pkgs/pull/82170#issuecomment-1265058628

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> I am just a "private" person trying to update the manifest for DDM so that it will actually work and not just be a dead link.

Do `@microsoft-github-policy-service agree` which is for individuals.

**Comment by DanHSO** [comment]
@microsoft-github-policy-service agree

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**